pages/resume.tsx

```diff
@@ -1,17 +1,17 @@
 import React, { FC } from 'react';
-import Resume from '../components/resume/resume';
+import { default as ResumeComponent } from '../components/resume/resume';
 import { Basics, Job } from '../components/resume/types';
 import resumeJson from '../data/resume/resume.json';
 import { ResumeJson, WorkItem } from '../data/resume/types';
 
-interface ResumePageProps {
+interface ResumeProps {
     basics: Basics;
     skills: string[];
     jobs: Job[];
 }
 
-const ResumePage: FC<ResumePageProps> = uiModel => {
-    return <Resume {...uiModel} />;
+const Resume: FC<ResumeProps> = uiModel => {
```

Comment:
Renamed the component name to be used as the page title. This isn't foolproof and likely will need to be redone in the future as more pages are added, but this is a quick way to accomplish a better tab title.

The other con to this is that component name serves a different purpose compared to tab title. Ideally this page title would be a defined property, perhaps as part of the a route definition. Going with the simple approach for now due to less effort and may reevaluate the decision in the future if complexity grows or intent further diverges.